frontend/src/components/common/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,52 @@
+import { Icon, IconifyIcon } from '@iconify/react';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export interface ActionButtonProps {
```

Comment:
It's not clear to me what an ActionButton does. If this is about adding a tooltip, maybe we can override the IconButton and Button and receive a tooltip prop instead?